src/main/java/lotto/LottoApplication.java

```diff
@@ -0,0 +1,40 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import lotto.domain.Lotto;
+import lotto.domain.LottoChecker;
+import lotto.domain.LottoNumberGenerator;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoApplication {
+
+  private static final int LOTTO_PRICE = 1000;
+
+  public static void main(String[] args) {
+    Scanner scanner = new Scanner(System.in);
+
+    int purchaseAmount = InputView.getPurchaseAmount(scanner);
+    int ticketCount = purchaseAmount / LOTTO_PRICE;
+
+    List<Lotto> tickets = new ArrayList<>();
+
+    for (int i = 0; i < ticketCount; i++) {
+      tickets.add(new Lotto(LottoNumberGenerator.getInstance().generate()));
+    }
+
+    ResultView.showLottos(tickets);
+
+    List<Integer> winningNumbers = InputView.getWinningNumbers(scanner);
+
+    Map<Integer, Long> lottoResults = LottoChecker.calculateResults(tickets, winningNumbers);
+    double profitRate = LottoChecker.calculateProfitRate(lottoResults, purchaseAmount);
+    ResultView.printResults(lottoResults, profitRate);
```

Comment:
```suggestion
    LottoResult result = LottoChecker.execute(tickets, winningNumbers, purchaseAmount);
    ResultView.printResults(result);
```
- LottoChecker에게 로또 결과을 판단하는 역할이 위임된 것으로 보이는데요. Public Operation은 하나여도 충분하지 않을까요?
- primitive type으로 응답을 받아 인자로 각각 전달하기 보다, Value Object를 도출해보는 건 어떨까요